environments/python/builder/defaultBuildCmd

```diff
@@ -1,2 +1,6 @@
 #!/bin/sh
-pip3 install -r ${SRC_PKG}/requirements.txt -t ${SRC_PKG} && cp -r ${SRC_PKG} ${DEPLOY_PKG}
\ No newline at end of file
+if [ -f ${SRC_PKG}/requirements.txt ]
```

Comment:
add `set -euxo pipefail` let the build failed if any of error occurs

Reply:
Done @life1347